frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -278,6 +278,20 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                                                                 Add open-ended choice
                                                             </LemonButton>
                                                         )}
+                                                        <LemonField name="shuffleOptions" className="mt-2">
```

Comment:
for completeness sake + learning this part of our tooling, could you add a snapshot of the multiple select question option? We control that via storybook -> https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/surveys/Surveys.stories.tsx#L218 -> https://storybook.posthog.net/?path=/docs/hello--docs

Reply:
done! I added a new survey with multiple options.